project/BinaryIncompatibilities.scala

```diff
@@ -20,6 +20,18 @@ object BinaryIncompatibilities {
   )
 
   val Linker = Seq(
+      // !! SemVer-Minor Breaking
```

Comment:
Maybe adjust this to make it clear that this is according to our compatibility guarantees (so OK).